src/app/faq/faq.component.sass

```diff
@@ -0,0 +1,41 @@
+@import './src/components/variables.sass'
+
+.faq
+    display: grid
+    grid-gap:20px
+
+.faq p
+    font-size: 40px
+    line-height:2.75rem
```

Comment:
Also, we probably don't want to use px and rem interchangeably. 

A good note for font-size and line-height is that line-height should always be at least 2px larger than the font-size